swift-ticketing/test/swift_ticketing/client.clj

```diff
@@ -1,22 +1,35 @@
 (ns swift-ticketing.client
```

Comment:
Is a client inside a test considered a good practice for testing in Clojure?
@zzat Can you point me to any blogs where you have found this?
Also, did you consider using ring.mock to make these requests?